frontend/src/Status.jsx

```diff
@@ -22,10 +22,10 @@ const headers = new Headers({
 });
 
 // Valid browser other than IE
-const VALID_BROWSERS = ["chrome", "firefox"];
-const IE_STRING = "IE";
+export const VALID_BROWSERS = ["chrome", "firefox"];
```

Comment:
why are these two in an array, and IE in a string? It makes where you import it a bit hard to read.

Reply:
This is because of the way that the _validateBrowser()_ logic has been created in _Status.jsx_ file.

![image](https://user-images.githubusercontent.com/23021242/53885644-442ad400-3fec-11e9-83f3-e39c3b9d39fd.png)

Do you want me to add '**IE**' string in **'BROWSER_STRING'** and update the logic of validateBrowser__?

Reply:
It's fine to leave for now, but maybe, we'll refactor if we have to do more browser based work in the future.